troposphere/static/js/collections/SizeCollection.js

```diff
@@ -9,7 +9,7 @@ define(function (require) {
   return Backbone.Collection.extend({
     model: Size,
 
-    url: globals.API_V2_ROOT + "/sizes",
+    url: globals.API_V2_ROOT + "/sizes?archived=true",
```

Comment:
Okay - so the proper way to do this is exactly what Steve noted, with `fetchWhere`. 

If you poke around the codebase, you'll see that `stores.SizeStore` has some occurrences of this to limit the sizes to those associated/related to a cloud-provider: 

```
troposphere/static/js/components/providers/Instances.react.js:        sizes = stores.SizeStore.fetchWhere({
troposphere/static/js/components/providers/Instances.react.js-          provider__id: provider.id,
troposphere/static/js/components/providers/Instances.react.js-          page_size: 100
troposphere/static/js/components/providers/Instances.react.js-        }),
--
troposphere/static/js/components/providers/Resources.react.js:        sizes = stores.SizeStore.fetchWhere({
troposphere/static/js/components/providers/Resources.react.js-          provider__id: provider.id,
troposphere/static/js/components/providers/Resources.react.js-          page_size: 100
troposphere/static/js/components/providers/Resources.react.js-        });
--
troposphere/static/js/components/providers/Stats.react.js:        sizes = stores.SizeStore.fetchWhere({
troposphere/static/js/components/providers/Stats.react.js-          provider__id: provider.id,
troposphere/static/js/components/providers/Stats.react.js-          page_size: 100
troposphere/static/js/components/providers/Stats.react.js-        });
```

So, this scopes the need for changed to the following files: 
- troposphere/static/js/components/providers/Instances.react.js
- troposphere/static/js/components/providers/Resources.react.js
- troposphere/static/js/components/providers/Stats.react.js